ZIGSIMPlus/Presenters/CommandDataSelectionPresenter.swift

```diff
@@ -45,6 +45,12 @@ final class CommandDataSelectionPresenter: CommandDataSelectionPresenterProtocol
             fatalError("Invalid CommandData selected: \(labelString)")
         }
         AppSettingModel.shared.isActiveByCommandData[label]?.toggle()
+
+        // Update commands availability
```

Comment:
This comment should clarify the reason of running the same process as `init()`.
(Actually I didn't understand it.)

Reply:
We need to update commandData every time users select/deselect commands, because "command.isAvailable" may change in some commands.
In this case, `Face Tracking` must be disabled when the user selects `ARKit`.